src/std/10-std-builder.md

```diff
@@ -1 +1,175 @@
 # Builder
+
+The **builder** module defines a `builder` type that allows to build and install C++ modules to be used with Feral
```

Comment:
`builder_t` type

~~install C++ modules~~ install external modules
(because it's not C++ exclusive, it can install Feral only modules too)